migtests/lib/yb.py

```diff
@@ -130,6 +130,16 @@ def fetch_datatypes_of_all_tables_in_schema(self, schema_name="public") -> Dict[
 			tables[table_name].append(data_type)
 		return tables
 
+	def fetch_datatypes_and_columns_of_all_tables_in_schema(self, schema_name="public") -> Dict[str, Dict[str,str]]:
```

Comment:
Rename: `get_column_to_data_type_mapping(...)`.

Can we reduce the duplication between `fetch_datatypes_of_all_tables_in_schema()` and this function.

Reply:
Changed the function name

We could actually change the validation part of oracle datatype testing to use the new `get_column_to_data_type_mapping()`. That way we wont need `fetch_datatypes_of_all_tables_in_schema()` at all.

Reply:
Correct. Would you like to give it a try in a separate PR?